tests/installation/disable_grub_timeout.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Disable grub timeout from the Installer
```

Comment:
You are describing what you do but can you also state why?

Reply:
fixed

Reply:
you stating that it is fixed , but are sure that you pushed changes ?

Reply:
yes, I believe that when the change is in a comment I think github does not detect it. line #11 has the info required.